src/api/app/models/report.rb

```diff
@@ -38,6 +38,10 @@ def event_parameters
     { id: id, reporter: reporter.login, reportable_id: reportable_id, reportable_type: reportable_type, reason: reason, category: category }
   end
 
+  def other_reports_from_reportable
+    reportable.reports - [self]
```

Comment:
Codecov wishes you add a little test for this in spec/models/report_spec.rb :thinking:

Reply:
Alright, give me some minutes to add it